lib/ansible/modules/network/checkpoint/checkpoint_network_facts.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
```

Comment:
```suggestion
                    'supported_by': 'community'}
```

Reply:
@justjais Are u sure about that? 
that's how Riki wrote in all modules..

Reply:
@justjais

Reply:
@chkp-orso yes, since Ricky was part of Anisble networking and he wrote the modules, so it'll be supported by `network` and since the new CP modules shall be contributed by u it shall be `community`

Reply:
@justjais But as Riki once told us, he just wrote a quick example, the modules he wrote are not good enough from checkpoint aspect, so anyway I would prefer to delete and rewrite all his modules..   
Is it possible?

Reply:
@chkp-orso @justjais quick recap of the previous conversations we had. Check Point expressed the desire to take ownership of the modules RHT wrote as part of their certified content. More than "delete and rewrite" we discussed updating and aligning them to the others Check Point is working on. The current modules, already merged in 2.8, are in TP and are being used to test the technology and use cases. You can start submitting changes and updates to these modules as part of your takeover and users will be able to consume the latest version.
In terms of tagging Check Point's modules will start as "community" and then move to "certified" once the process is complete.

Reply:
@justjais So just to make sure I understand: it's not possible to delete and rewrite Rikis' modules in version 2.9?
only thing possible - to apply changes?

Reply:
@chkp-orso @justjais you can always write new modules to replace them and then deprecate the old ones in 2.9. Before doing that, I would recommend to evaluate if you can achieve what you are looking for through updates. That would probably be a better experience from a user standpoint.

Reply:
Thanks!, nevertheless I would prefer replace those files, because they are totally different.
So how can I upload those new files?

Reply:
@cross-logic @chkp-orso If u want to re-write the module post rigorous evaluation that the respective module cannot serve ur purpose then in that case you would need to make sure the changes are backwards compatible, e.g. don't remove module options or change the return values without a deprecation period.

Reply:
@justjais great thanks!